ansible/roles/gpu/tasks/nvidia-gpu.yaml

```diff
@@ -112,7 +112,8 @@
 - name: install driver
   shell:
     ./nvidia-installer -s \
-      --log-file-name=$(pwd)/nvidia-installer.log
+      --log-file-name=$(pwd)/nvidia-installer.log \
+      --no-cc-version-check
```

Comment:
Can you please add a comment explaining the new flag?

Reply:
yeah its to align with upstream image builder